packages/project-access/src/translations/project-access.i18n.json

```diff
@@ -0,0 +1,6 @@
+{
+    "error": {
+        "projectRootNotFound": "Could not find any project root for {{-path}}. Parameter 'sapuxRequired' was {{sapuxRequired}}",
```

Comment:
`sapuxRequired` is a property in your function that is not known to a human being using anything consuming this module, so I don't think that this is a proper error message that a user should ever see. And, if it isn't intended to be one, why does it need to be translated?

Reply:
I guess it is a bit of a grey zone. How would you classify the OUTPUT view in VSCode:
![image](https://user-images.githubusercontent.com/66327622/191281261-83be7743-c2e3-4fe8-a296-7d159e7b2988.png)

All texts in there used to be translated. You want to change this?

Reply:
This is logging information - quite technical information, so I would not expect it to be translated.

Reply:
@kranthie-sap tried using `Change Display Language` in the command palette to change the display language. It looks like VS Code also doesn’t translate the log messages in the output channels, so I'd recommend we align with VSCode here.

Reply:
removed usage of `i18next`